client/src/controllers/SwapController.ts

```diff
@@ -58,6 +59,7 @@ export class SwapController extends Controller<
   declare iconValue: string;
   declare loadingValue: boolean;
   declare srcValue: string;
+  declare reflectValue: boolean;
```

Comment:
Same here, can we keep these in alpha order.

So `reflect` would be before `src`.